src/search_renderer.cpp

```diff
@@ -90,6 +93,11 @@ std::string SearchRenderer::getHtml()
     std::ostringstream s;
     s << it.getZimId();
     result.set("resultContentId", mp_nameMapper->getNameForId(s.str()));
+    std::shared_ptr<zim::Archive> archive;
+    try {
+       archive = mp_library->getArchiveById(s.str());
+       result.set("bookTitle", archive ? getArchiveTitle(*archive) : "");
```

Comment:
An empty string for `bookTitle` won't disable the mustache block marked with `{{#bookTitle}}...{{/bookTitle}}`. You will need to pass a `kainjow::mustache::data(false)` instead.

Reply:
Updated that so it uses mustache's false.  Really this should be pertty much impossible. Both the `{#bookTitle}` wrapper and this null check are meant as guards against future bugs.